rpc/client.go

```diff
@@ -126,13 +127,20 @@ func (c *Client) nextSeq() uint32 {
 }
 
 func (c *Client) replyError(req *Message) {
+	stats.RemovePendingOp(req.ID, false)
 	delete(c.messages, req.Seq)
 	req.Type = TypeError
 	req.Data = []byte(c.err.Error())
 	req.Complete <- struct{}{}
 }
 
 func (c *Client) handleRequest(req *Message) {
+	switch req.Type {
```

Comment:
The RPC layer is all async, but higher level code like replica and control are all synchronous.  It makes more sense to annotate the code for this ops tracking at a higher level.  Then the code will be straight forward, `InsertPendingOp(); ReadAt(); RemovePendingOp()`